Servers/how-to-update-softnas.md

```diff
@@ -0,0 +1,188 @@
+{{{
+  "title": "SoftNAS: How to update",
+  "date": "11-11-2015",
+  "author": "Maxim Volkov",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
```

Comment:
Please code in html or markdown not a mix. Recommend markdown.  Update your entire kb